cmd/podman/pods/logs.go

```diff
@@ -0,0 +1,127 @@
+package pods
+
+import (
+	"os"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/validate"
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// logsOptionsWrapper wraps entities.LogsOptions and prevents leaking
+// CLI-only fields into the API types.
+type logsOptionsWrapper struct {
+	entities.PodLogsOptions
+
+	SinceRaw string
+
+	UntilRaw string
+}
+
+var (
+	logsOptions     logsOptionsWrapper
+	logsDescription = `Displays logs for pod with one or more containers.`
+	logsCommand     = &cobra.Command{
+		Use:               "logs [options] POD",
+		Short:             "Fetch logs for pod with one or more containers",
+		Long:              logsDescription,
+		Args:              cobra.ExactArgs(1),
+		RunE:              logs,
+		ValidArgsFunction: common.AutocompletePods,
+		Example: `podman logs podID
+		podman pod logs -c ctrname podName
+		podman pod logs --tail 2 mywebserver
+		podman pod logs --follow=true --since 10m podID
+		podman pod logs mywebserver`,
+	}
+
+	containerLogsCommand = &cobra.Command{
+		Use:               logsCommand.Use,
+		Short:             logsCommand.Short,
+		Long:              logsCommand.Long,
+		Args:              logsCommand.Args,
+		RunE:              logsCommand.RunE,
+		ValidArgsFunction: logsCommand.ValidArgsFunction,
+		Example: `podman pod logs podId
+		podman pod logs -c ctrname podName
+		podman pod logs --tail 2 mywebserver
+		podman pod logs --follow=true --since 10m podID`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: logsCommand,
+	})
+	logsFlags(logsCommand)
+	validate.AddLatestFlag(logsCommand, &logsOptions.Latest)
```

Comment:
The logs command is already used for container. I don't think you can reuse this for pod logs.

Reply:
@Luap99 resolved this in latest commit.